src/rules/newline-after-import.js

```diff
@@ -0,0 +1,52 @@
+/**
+ * @fileoverview Rule to enforce new line after import not followed by another import.
+ * @author Radek Benkel
+ */
+
+//------------------------------------------------------------------------------
+// Rule Definition
+//------------------------------------------------------------------------------
+
+function getLineDifference(node, nextToken) {
+  return nextToken.loc.start.line - node.loc.start.line
+}
+
+function ensureNoForbiddenKeyword(context, node, tokenToInspect, tokenValue) {
+  if (getLineDifference(node, tokenToInspect) === 1
+    && tokenToInspect.type === 'Keyword' && tokenToInspect.value !== tokenValue)
+  {
+    context.report({
+      loc: tokenToInspect.loc.start,
+      message: 'Expected empty line after import/require ' +
+        'statement not followed by another import.',
+    })
+  }
+}
+
+module.exports = function (context) {
+  return {
+    ImportDeclaration: function (node) {
+      const nextToken = context.getSourceCode(node).getTokenAfter(node)
+
+      if (!nextToken) {
+        return
+      }
+
+      ensureNoForbiddenKeyword(context, node, nextToken, 'import')
+    },
+    CallExpression: function(node) {
+      if (node.callee.name === "require") {
+        const nextTokens = context.getSourceCode(node).getTokensAfter(node, 2)
+        const tokenToInspect = nextTokens.length > 1 && nextTokens[0].type === 'Punctuator'
+          ? nextTokens[1]
+          : nextTokens[0]
+
+        if (!tokenToInspect) {
+          return
+        }
```

Comment:
You could move this check to `ensureNoForbiddenKeyword` for both `import` and `require` cases.